### DISCUSSION TITLE
Inttelij when i give commit option files selection option not appearing  changes there and also colour changed

### DISCUSSION THREAD

**Discussion by palsachennaiah** [learndevopsonline/learndevopsonline#1609] [Q&A]
![Screenshot 2023-11-16 122716](https://github.com/learndevopsonline/learndevopsonline/assets/148023299/c55438ad-67e9-418d-a27f-693e95367e16)
![Screenshot 2023-11-16 123241](https://github.com/learndevopsonline/learndevopsonline/assets/148023299/6b8910be-463e-4562-9b55-69b20a9a07be)

**Comment by sreedharm07** [comment]
if there is any change it will show 
if there is no change it wont show

**Reply by palsachennaiah** [reply]
I changed but it did not show

**Reply by palsachennaiah** [reply]
Do you observe left side files background colour how is it.

**Reply by sreedharm07** [reply]
@palsachennaiah 
it will show 
if not 
click on main you can find push option then select what ever file you want to push
![image](https://github.com/learndevopsonline/learndevopsonline/assets/137472068/4ebdabd7-7137-4d65-a68b-ac91e8e72749)

**Reply by palsachennaiah** [reply]
I have done that but not appearing, like which mention above in my inttelij not shows

**Reply by sreedharm07** [reply]
ok i said what i know, actually it will works but not for you, so please join lab session.
or
do trail and error methods.
